frontend/src/metabase/search/components/UserNameDisplay/UserNameDisplay.tsx

```diff
@@ -0,0 +1,37 @@
+import { t } from "ttag";
+import { useUserListQuery } from "metabase/common/hooks/use-user-list-query";
+import { Text } from "metabase/ui";
+import type { UserId } from "metabase-types/api";
+
+export type UserNameDisplayProps = {
+  value: UserId | null;
```

Comment:
`userId` feels more appropriate since this is not an input component:

```suggestion
  userId: UserId | null;
```

Alternatively this component could be named `UserNameValue` and then `value` prop would make sense.

Reply:
Renamed to `userId`